resources/ts/Components/Navigation.tsx

```diff
@@ -24,6 +24,19 @@ const Link = (props: ComponentProps<"a">) =>
     <RouterLink {...(props as InertiaLinkProps)} />
   );
 
+const allPages: Page[] = [
+  {
+    component: "Integrations/Index",
+    key: "integrations",
+  },
+  { key: "opportunities" },
+  {
+    key: "prices",
+  },
+  { key: "documentation" },
+  { component: "Support/Index", key: "support" },
+] as const;
+
```

Comment:
Would it be an idea to add a property `visibility`: ['authenticated', 'notAuthenticated']  so you can filter on them? 

```suggestion
const allPages: Page[] = [
  {
    component: "Integrations/Index",
    key: "integrations",
    visibility: ['authenticated']
  },
  { key: "opportunities", visibility: ['notAuthenticated'] },
  {
    key: "prices",
    visibility: ['notAuthenticated']
  },
  { key: "documentation",  visibility: ['notAuthenticated'] },
  { component: "Support/Index", key: "support",  visibility: ['authenticated', 'notAuthenticated'] },
] as const;

```